helm/values-dev.yaml

```
@@ -92,7 +92,9 @@ microservice-chart:
     APPLICATIONINSIGHTS_CONNECTION_STRING: 'app-insight-connection-string'
     OTEL_EXPORTER_OTLP_HEADERS: elastic-apm-secret-token
     MONGODB_CONNECTION_URI: 'notices-mongo-connection-string'
-    TEMPLATE_STORAGE_CONN_STRING: 'templates-storage-account-connection-string'
+    TEMPLATE_STORAGE_CONN_STRING: 'templates_storage_account_connection_string'
```

use `-` instead of `_`
```suggestion
    TEMPLATE_STORAGE_CONN_STRING: 'templates-storage-account-connection-string'
```